src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -43,6 +43,19 @@ const pageQuery = gql`
 `;
 
 export default {
+	metaInfo() {
+		this.retrieveMetaInfo();
+		return {
+			title: this.meta.title,
+			meta: [
```

Comment:
We should also add the meta tags for `twitter:description` and `og:description` with the same value as `description`

Reply:
That's a good point! Should we do the same for `twitter:title` and `og:title`, setting them to the same value as `title`?

Reply:
Yep, lets add titles as well

Reply:
Good catch @eddieferrer ! I'll add those meta tags